heka/sandbox/filters/firefox_monthly_dashboard.lua

```diff
@@ -91,7 +91,7 @@ local function update_month(ts, cid, day_changed)
         current_month = month
     end
 
-    local country = fx.get_country_id(read_message("Fields[geo]"))
+    local country = fx.get_country_id(read_message("Fields[country]"))
```

Comment:
Do we need to `s/geo/country/` on line 136 too? or is it called "geo" for compatibility?